### PR TITLE
Fix for issue https://github.com/bartdag/py4j/issues/278

### PR DESCRIPTION
**Pull request by scottslewis**
PythonProxyHandler class (java).  Also enhanced java_callback_test.py to
test.  Without fix to PythonProxyHandler new test fails, with fix
succeeds.  see IntegrationTest.testProxyReturnerNull

### CODE REVIEW

File: py4j-python/src/py4j/tests/java_callback_test.py [hunk 1]
```diff
@@ -318,6 +321,13 @@ def testProxyReturnerVoid(self):
         output = example.computeNothing(returner)
         self.assertEqual(output, 1)
 
+    def testProxyReturnerNull(self):
+        sleep()
+        example = self.gateway.jvm.py4j.examples.ReturnerExample()
+        returner = Returner()
+        output = example.computeNull(returner)
+        self.assertEqual(output, None)
```
Comment (bartdag) [inline py4j-python/src/py4j/tests/java_callback_test.py]: I was about to suggest to use assertIsNone, but it doesn't exist in Python 2.6, so assertEqual it is!

**Comment by bartdag** [comment]
Thanks for the pull request, it is almost good to go :-) Can you add your name to AUTHORS.txt?

**Comment by scottslewis** [comment]
Hi @bartdag .   OK added myself to authors.txt with second commit for this pull request.   Thanks for merging this quickly, it will be nice to have in 0.10.6.   

What's your thinking on timing for 0.10.6?   

Would it be possible to deal with:  https://github.com/bartdag/py4j/issues/243  ?   I could help with impl 
 and testing if you like.   The reason this is important is that for my use case there may be cases when there may be more than one gateway, and each gateway should have it's own classloading strategy (since there is more than one classloader in OSGi).

**Comment by bartdag** [comment]
Regarding #243, do not hesitate to open a pull request. I can take the work from there (e.g., port the fix to ClientServer). I don't think I'll have time to work on it in the coming weeks otherwise.

My focus for 0.10.6 is #275 

I should be able to work on it next Friday. If I can complete it, I'll be able to release 0.10.6 during the next weekend.

Thanks for your pull request! I've squashed and merge it.

**Pull request closed without merging**